test/zkbob/ZkBobPoolETH.t.sol

```diff
@@ -0,0 +1,580 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity 0.8.15;
+
+import "forge-std/Test.sol";
+import "@openzeppelin/contracts/token/ERC721/presets/ERC721PresetMinterPauserAutoId.sol";
+import "@uniswap/v3-periphery/contracts/interfaces/INonfungiblePositionManager.sol";
+import "@uniswap/v3-core/contracts/libraries/TickMath.sol";
+import "../shared/Env.t.sol";
+import "../mocks/TransferVerifierMock.sol";
+import "../mocks/TreeUpdateVerifierMock.sol";
+import "../mocks/BatchDepositVerifierMock.sol";
+import "../mocks/DummyImpl.sol";
+import "../../src/proxy/EIP1967Proxy.sol";
+import "../../src/zkbob/ZkBobPoolETH.sol";
+import "../../src/zkbob/ZkBobDirectDepositQueue.sol";
+import "../../src/BobToken.sol";
+import "../../src/zkbob/manager/MutableOperatorManager.sol";
+import "../shared/ForkTests.t.sol";
+import "@uniswap/v3-periphery/contracts/interfaces/external/IWETH9.sol";
+import "../../src/zkbob/manager/kyc/SimpleKYCProviderManager.sol";
+import "../../src/zkbob/ZkBobDirectDepositQueueETH.sol";
+
+contract ZkBobPoolETHTest is AbstractMainnetForkTest {
```

Comment:
Seems like a lot of tests actually remain the same, maybe we can keep all the tests in the single file, just add a couple of ETH-specific ones OR put shared tests in the inherited base contract.